.github/workflows/ci-checks.yml

```diff
@@ -0,0 +1,29 @@
+name: Flutter Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Do we have any dependency on v1? if not can we use the latest ones?

Reply:
I think we don't have any dependency. lets check and update v3